http/logical.go

```diff
@@ -75,6 +75,30 @@ func buildLogicalRequest(core *vault.Core, w http.ResponseWriter, r *http.Reques
 		}
 	}
 
+	// If we are a read operation, try and parse any parameters
+	if op == logical.ReadOperation {
+		getData := map[string]interface{}{}
+
+		for k, v := range r.URL.Query() {
+			// Skip the help key as this is a reserved parameter
+			if k == "help" {
```

Comment:
`list` is also a reserved parameter (you can use `list=true` instead of the verb `LIST`)

Reply:
This is already special cased above: https://github.com/hashicorp/vault/blob/f83f41436d0f3f391adabc80c513abe49664c5e2/http/logical.go#L37-L49

Reply:
Oops, missed that...didn't expand the pre-context enough.